### PR TITLE
Add missing fixture

### PR DESCRIPTION
**Pull request by mefmund**
Add missing fixture in argument list.

### CODE REVIEW

File: doc/en/fixture.rst [hunk 1]
```diff
@@ -879,7 +879,7 @@ Here's what that might look like:
         admin_client.delete_user(user)
 
 
-    def test_email_received(receiving_user, email):
+    def test_email_received(sending_user, receiving_user, email):
         email = Email(subject="Hey!", body="How's it going?")
         sending_user.send_email(_email, receiving_user)
```
Comment (The-Compiler) [inline doc/en/fixture.rst:884]: Unrelated to your fix, but I think this should also be

```suggestion
        sending_user.send_email(email, receiving_user)
```

for this to actually work, no?

Comment (mefmund) [reply doc/en/fixture.rst:884]: Yes, should I commit your suggestion?

Comment (RonnyPfannschmidt) [reply doc/en/fixture.rst:884]: i beleive a key issue of why this example bit-rotted/was incorrect to begin with, is that its not part of the regendoc process (where we update the example outputs in the docs by running the examples)

as  a followup/extension we might want to add a actual testrun of this stuff

Comment (nicoddemus) [reply doc/en/fixture.rst]: > Yes, should I commit your suggestion?

In general yes, but I did take the liberty of going ahead and doing it myself, thanks. 👍

Comment (nicoddemus) [reply doc/en/fixture.rst]: > is that its not part of the regendoc process 

I agree, created https://github.com/pytest-dev/pytest/issues/8213. 👍

**Comment by nicoddemus** [comment]
Thanks @mefmund!

**Pull request merged**